### DISCUSSION TITLE
Do the .`rpm` packages support OpenSUSE TW?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [evilsocket/opensnitch#1033] [Q&A] [answered]
Unlike https://github.com/evilsocket/opensnitch/discussions/595#discussion-3819378, I don't have an issue with this currently. I just want to know whether the `.rpm` packages are tested on cpe:/o:opensuse:tumbleweed. If not, I can probably use https://github.com/evilsocket/opensnitch/discussions/1035#discussion-5584818 to add support eventually.

**Comment by gustavo-iniguez-goya** [comment, marked as answer]
Hi @rokejulianlockhart ,

Latest releases (v1.6.x) have been tested on Leap. I've just tested it on TW and works fine.
